python/llm/example/GPU/HuggingFace/LLM/aquila/README.md

```diff
@@ -77,7 +77,6 @@ export ENABLE_SDP_FUSION=1
 
 ```bash
 export SYCL_CACHE_PERSISTENT=1
-export BIGDL_LLM_XMX_DISABLED=1
```

Comment:
We may merge Intel iGPU & Intel Arc A-Series as one section now, as they share the same env variables